airbyte-integrations/connectors/source-nytimes/source_nytimes/spec.yaml

```diff
@@ -0,0 +1,44 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/nytimes
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Nytimes Spec
+  type: object
+  required:
+    - api_key
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      title: API Key
+      description: API Key
+      airbyte_secret: true
+      order: 0
+    year:
+      type: integer
+      title: Year
+      description: Year
+      minimum: 1851
+      order: 1
+    month:
+      type: integer
+      title: Month
+      description: Month
+      minimum: 1
+      maximum: 12
+      order: 2
+    period:
+      type: integer
+      title: Period
+      description: Period
+      order: 3
+      enum:
+        - 1
+        - 7
+        - 30
```

Comment:
Please make clear this is used only for most popular endpoints.

Reply:
Noted!